src/index.js

```diff
@@ -3,16 +3,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```

Comment:
- [ ] You have a folder created that you do not need, kindly remove it 
![image](https://user-images.githubusercontent.com/62705947/189132382-bd81156f-e8c6-465f-a1e4-efdb28e774b4.png)

Reply:
- [ ] Remove these images, they will not be needed 
![image](https://user-images.githubusercontent.com/62705947/189132712-646be7b5-03c4-4c12-a6cd-8ef710fc7a8e.png)

Reply:
@Kwaleyela-Ikafa sorry but they are project requirements  
![image](https://user-images.githubusercontent.com/62705947/189135376-e4ea4093-7ec7-4121-a17b-841e217adcc0.png)

Reply:
I don't know how i forgot them there, appreciate the review though